imports/ui/Results.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { Component } from 'react';
+import "./css/Results.css"; // css files
+import SearchBar from './SearchBar.jsx';
+import ResultsDisplay from './ResultsDisplay.jsx';
+
+
+/*
+  Results Component. Short description if needed.
+
+  Identify as one of the following components:
+  Simple styling: mainly renders HTML and CSS,
+  Container: combines multiple components into a single feature
+  View: top-level component accessed by a URL endpoint defined by the router in main.jsx
+
+  Include a breif description of the component's purpose, where it falls in the
+  component tree, and any inportant information it accesses or modifies.
+  Include the route for View components.
+*/
+
+export class Results extends Component {
+  constructor(props) {
+    super(props);
+    _isMounted = false;
```

Comment:
Variable not created via `var`, `let`, or `const` will be global variable!
You should avoid this!